iOS_SDK/OneSignalSDK/OneSignalUser/Source/Executors/OSUserExecutor.swift

```diff
@@ -190,6 +190,20 @@ extension OSUserExecutor {
         executePendingRequests()
     }
 
+    /**
+     This Create User call expects an Identity Model with external ID to hydrate the OneSignal ID
+     */
+    static func createUser(identityModel: OSIdentityModel) {
+        guard identityModel.externalId != nil else {
+            OneSignalLog.onesignalLog(.LL_ERROR, message: "createUser(identityModel) called with missing external ID")
```

Comment:
If we hit this case what happens? Will we try again in the same session?

Reply:
The purpose of this version of `ceateUser()` is to get an OSID for an EUID. The EUID must exist because the only usage of this method is after an Identify User 409 conflict, in which case we would have the EUID. The guard and error log is more for noticing that we do something wrong such as using this method inappropriately down the line. 

Thinking about this again, it may be clearer to make EUID a parameter instead of it being implicit.

Reply:
Ya it should probably require it as a param. Would it retry next session?

Reply:
I edited my response after to talk about retrying. This call would be made if the user has changed since, so this would be for a previous user. It won't retry since no EUID, no way to get its OSID. That means if there are pending requests for this previous user, they will effectively be dropped.